azkaban-common/src/main/java/azkaban/jobExecutor/AbstractProcessJob.java

```diff
@@ -45,10 +45,13 @@
   private static final String SENSITIVE_JOB_PROP_VALUE_PLACEHOLDER = "[MASKED]";
   private static final String JOB_DUMP_PROPERTIES_IN_LOG = "job.dump.properties";
 
-  private final String jobPath;
+  //Notes: These variables will be accessed directly throw inherited classes,
+  //       which are out of this package. Please remain them to be protected
+  protected final String _jobPath;
   protected String cwd;
```

Comment:
Please change to _cwd, in case any separate jobtype MPs are using it.

Reply:
good catch. copied.